api/Window.json

```diff
@@ -1371,6 +1373,59 @@
           }
         }
       },
+      "console": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Window/console",
+          "spec_url": "https://heycam.github.io/webidl/#dfn-namespace-object",
```

Comment:
This isn't the right spec URL IMHO, it should be https://console.spec.whatwg.org/#console-namespace

Web IDL's definition of namespaces explains why that ends up as a property on the global object, but that's true of interfaces too.

And speaking of the global object, the console object is available in workers too, so should this really be on Window?

Reply:
I'll update the spec_url.

Wrt `Window.console`, `Window` is the global scope for non-worker case. As it is available in workers, it means we probably should also have a WorkerGlobalScope.console entry.

But… I never encoutered namespaces before, so I've never thought about it. Maybe we should open a follow-up issue for console in workers? (I am also fine to remove it from this PR)

Reply:
Backed out.